src/utils/schemaUtils.js

```diff
@@ -33,7 +33,14 @@ function getAllResponseContentTypes(responses) {
  * @param {string} omitByValue omit if the prop value equals to this value
  */
 function omitPropsFromSchema(dereferencedSchema, omitByPropName, omitByValue) {
-    if (dereferencedSchema.type === 'object') {
+    const combinedSchemaType = getCombinedSchemaType(dereferencedSchema);
```

Comment:
I'd be glad for suggestions for a better name for `combinedSchemaType` and `getCombinedSchemaType`